_troubleshooting/slow-tiering-chunks.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Slow tiering of chunks
+section: troubleshooting
+products: [cloud]
+topics: [data tiering]
+keywords: [data tiering]
+tags: [data tiering]
+---
+
+
+<!---
+* Use this format for writing troubleshooting sections:
+ - Cause: What causes the problem?
+ - Consequence: What does the user see when they hit this problem?
+ - Fix/Workaround: What can the user do to fix or work around the problem? Provide a "Resolving" Procedure if required.
+ - Result: When the user applies the fix, what is the result when the same action is applied?
+* Copy this comment at the top of every troubleshooting page
+-->
+
+Chunks are tiered asynchronously with an external "side-car" process. Chunks are selected to be tiered to s3 one at a time ordered by their enqueue time.
+
+To see the number of remaining chunks  to be tiered query the `_osm_tables.data_tier_queue` table. 
+
+```sql
+select count(*) from _osm_tables.data_tier_queue
```

Comment:
Not sure we want to document our internal details of implementation (_osm_internal_ schema)

Reply:
Yeah. I was not sure about this.

Reply:
Hello folks, we're trying to help to move on some stuck PRs here and I'd love to know if you end up in some agreement here as it seems closer to be merged.

As a Developer Advocate my preference is show all details as we can and avoid overwhelming user by wrapping more detailed queries into foldable components that can be expanded during the learning process. More detailed material we have, less support tickets we have as most of people look carefully our docs before opening a new issue.

Reply:
@jonatas We do not expose internal tables/schema etc because these can change between releases. The definition is not stable. 
We now have a view that shows this info.